roles/mysql/tasks/main.yml

```diff
@@ -30,8 +35,28 @@
     update_cache: true
   when:
     - mysql_server
-  notify:
-    - set root password
+
+- name: MySQL | Restart MySQL
+  ansible.builtin.service:
+    name: mysql
+    state: restarted
+  when: install_app_key.changed
+
+- name: MySQL | mysql_root_password
+  community.mysql.mysql_user:
+    login_unix_socket: "{{ mysql_socket }}"
+    login_user: root
+    login_host: localhost
+    login_password: ""
```

Comment:
What is the difference between the `login_password` and the `mysql_root_password` below? Null passwords make me uneasy.

Reply:
when you install mysql on Ubuntu it has no password for root. So we are saying log into mysql with your blank password then set the password to `mysql_root_password`